ros2topic/ros2topic/verb/info.py

```diff
@@ -40,5 +70,10 @@ def main(self, *, args):
                 return "Unknown topic '%s'" % topic_name
             type_str = topic_types[0] if len(topic_types) == 1 else topic_types
             print('Type: %s' % type_str)
-            print('Publisher count: %d' % node.count_publishers(topic_name))
-            print('Subscriber count: %d' % node.count_subscribers(topic_name))
+            print('\nPublisher count    : %d' % node.count_publishers(topic_name))
```

Comment:
nitpick: it looks odd to me with the added whitespace before the colon, especially when `Type:` does not have the same alignment. IMO, it looks fine without the added whitespace.

Reply:
Removed the whitespace.